terraform/gcp/modules/kms/kms.tf

```diff
@@ -53,3 +53,22 @@ resource "google_kms_key_ring_iam_member" "rekor_sa_kms_iam" {
   member      = format("serviceAccount:%s-rekor-sa@%s.iam.gserviceaccount.com", var.cluster_name, var.project_id)
   depends_on  = [google_kms_key_ring.rekor-keyring]
 }
+
+resource "google_kms_key_ring" "fulcio-keyring" {
+  name       = var.name
```

Comment:
This currently maps to `rekor-keyring` in the [variables.tf](https://github.com/sigstore/scaffolding/blob/main/terraform/gcp/modules/kms/variables.tf#L37-L41) file. You might want to create a new keyring called `fulcio-keyring`

Reply:
done